doc/abyss-pe.1

```diff
@@ -49,7 +49,7 @@ files containing single-end reads
 maximum number of branches of a bubble [2]
 .TP
 .B b
-maximum length of a bubble (bp) [10000]
+maximum length of a bubble (bp) [3*k]
```

Comment:
Let's just document the truth and say
`maximum length of a bubble (bp) [3*k for ABYSS and 10000 for PopBubbles]`
Done.